app/pages/403.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {Row, Col} from 'react-bootstrap';
+import DefaultLayout from 'layouts/default-layout';
+import Link from 'next/link';
+import getConfig from 'next/config';
+
+export default () => {
+  const feedbackUrl = getConfig()?.publicRuntimeConfig.FEEDBACK_SITE_URL;
+
+  return (
+    <>
+      <DefaultLayout session={null}>
+        <Row className="justify-content-center" style={{paddingTop: '3em'}}>
+          <Col md={{span: 6}} style={{textAlign: 'center'}}>
+            <h1>Something went wrong</h1>
+            <p>
+              If you got here from the registration page, this is an issue that
+              we are actively working to fix. We apologize for the
+              inconvenience.
+              <br />
+              You should be able to log in with the credentials you entered in
+              the registration form.
```

Comment:
Changing this:

> You should be able to log in with the credentials you entered in the registration form.

to:

> If you click "Login" above, you can now log in with the credentials you entered in the registration form.


...would make the suggested solution sound more definitive and tells them exactly what to do next (click Login).

Reply:
Otherwise, I think this is good to go!

Reply:
@kriscooke There might be other ways to end up on this page if keycloak fails. I updated the text following your recommendation